core/slim_sim.cpp

```diff
@@ -4933,13 +4933,8 @@ void SLiMSim::AddIndividualToTable(Individual *p_individual, table_collection_t
     }
 }
 
-void SLiMSim::WriteIndividualTable(table_collection_t *p_tables)
+void SLiMSim::AddCurrentGenerationToIndividuals(table_collection_t *p_tables)
 {
-    // This overwrites whatever might be previously in the individual table
-    individual_table_clear(p_tables->individuals);
-    // First prepend the existing remembered individuals
-    individual_table_copy(tables.individuals, p_tables->individuals);
-	
```

Comment:
This function is called [here](https://github.com/MesserLab/SLiM/blob/0e75d0155fc22d44ec6de8c2c355592702f2aa70/core/slim_sim.cpp), on output_tables, which has just been copied from tables. So these two lines (individual_table_clear and individual_table_copy) were unnecessary. What we want to do is add some more lines to the end of the existing individual table.

Maybe this makes more clear that we don't need to erase the existing `nodes.individual` column?